drainer/sync/kafka.go

```diff
@@ -248,3 +257,63 @@ func (p *KafkaSyncer) run() {
 		}
 	}
 }
+
+type ackItem struct {
+	ts   int64
+	item *Item
+	size int
+}
+
+type ackWindow struct {
+	win          []*ackItem
+	items        map[int64]*ackItem
+	unackedCount int
+	unackedSize  int
+}
```

Comment:
Can we add some comments on this work and why we need this structure?
Like:
put item in sarama.input
get item from sarama.success

Reply:
Added in 729fb6766b505c781d78eb0abd55f0144c6cbdbe.